packages/frontend-api/src/Resources/config/graphql-types/PriceDecorator.types.yml

```diff
@@ -0,0 +1,17 @@
+PriceDecorator:
+    type: object
+    config:
+        description: "Represents the price of the product"
+        fields:
+            priceWithVat:
+                type: "Money"
+                description: "Price with VAT"
+            priceWithoutVat:
+                type: "Money"
+                description: "Price without VAT"
+            vatAmount:
+                type: "Money"
+                description: "How much is going to build the new highway or agronomic subsidy"
```

Comment:
I believe this description should be more ... descriptive.
And I try to not use these jokes in prototypes 😄

Reply:
Well description of some field is one of places that are not reviewed with absolute focus, you got me!

Reply:
Not mor jokej, ok. https://github.com/shopsys/shopsys/pull/1471/commits/661bb034ecc5405d96f89658da1e28e5d98d2873